components/resources/brave_components_strings.grd

```diff
@@ -272,6 +272,8 @@
       <message name="IDS_BRAVE_UI_ALLOW_TIP" desc="">Allow tips on</message>
       <message name="IDS_BRAVE_UI_AMOUNT" desc="">Amount</message>
       <message name="IDS_BRAVE_UI_BACKUP" desc="">Backup</message>
+      <message name="IDS_BRAVE_UI_BACKUP_NOW" desc="">Backup Now</message>
```

Comment:
why is this needed as we only display notification in extensions?

Reply:
was an oversight removed